app/cypress/e2e/add-data-widget.cy.ts

```diff
@@ -328,4 +330,70 @@ describe('Add & Save Data Widget', () => {
       });
     });
   });
+
+  it('Test that save panel switches align with currently visible layers', () => {
+    openWidget('/community/dc/overview');
+
+    cy.findByRole('button', { name: 'Open Basemaps and Layers' }).click();
+    cy.findByRole('list', { name: 'Layer List' }).within(() => {
+      // toggle on a layer
+      cy.findByRole('switch', { name: 'Congressional Districts' }).click({
+        force: true,
+      });
+      // toggle off a layer
+      cy.findByRole('switch', { name: 'Boundaries' }).click({ force: true });
+    });
+
+    cy.get(adwId).within(() => {
+      cy.findByRole('tab', { name: 'Save' }).click();
+
+      cy.findByRole('switch', {
+        name: 'Toggle Congressional Districts',
+      }).should('have.attr', 'aria-checked', 'true');
+      cy.findByRole('switch', { name: 'Toggle Boundaries' }).should(
+        'have.attr',
+        'aria-checked',
+        'false',
+      );
+    });
+  });
+
+  it.only("Test that the save panel includes layers added from the widget's other tabs", () => {
```

Comment:
```suggestion
  it("Test that the save panel includes layers added from the widget's other tabs", () => {
```

Need to remove the `only` flag.